Arm/Memory/SeparateAutomation.lean

```diff
@@ -121,9 +121,18 @@ structure SimpMemConfig where
 structure Context where
   /-- User configurable options for `simp_mem`. -/
   cfg : SimpMemConfig
-
-def Context.init (cfg : SimpMemConfig) : Context where
-  cfg := cfg
+  /-- Cache of `bv_toNat` simp context. -/
+  bvToNatSimpCtx : Simp.Context
+  /-- Cache of `bv_toNat` simprocs. -/
+  bvToNatSimprocs : Array Simp.Simprocs
```

Comment:
Should we bundle these in a single struct, given that we basically always have these two types together? (Why are simprocs not just part of `Simp.Context` in the first place?)

Reply:
@alexkeizer I'm not sure why it's not part of `Simp.Context`, and that makes a little hesitant to bundle them together :)